pkgs/servers/mpd/default.nix

```diff
@@ -148,6 +148,11 @@ let
 
       depsBuildBuild = [ buildPackages.stdenv.cc ];
 
+      # FIXME: see if it stills necessary and remove on next release
+      patches = [
+        ./ffmpeg-6.1.patch
```

Comment:
I think we can use `fetchpatch` with https://github.com/MusicPlayerDaemon/MPD/commit/50c1e3738a672f933560247b5527d365d17b1c02